lidro/create_virtual_point/vectors/create_grid_2D_inside_maskhydro.py

```diff
@@ -6,14 +6,14 @@
 import shapely.vectorized
 
 
-def generate_grid_from_geojson(mask_hydro: gpd.GeoDataFrame, spacing: float, margin=0):
+def generate_grid_from_geojson(mask_hydro: gpd.GeoDataFrame, spacing=0.5, margin=0):
```

Comment:
pour garder le typehint, ut peux faire : 
```suggestion
def generate_grid_from_geojson(mask_hydro: gpd.GeoDataFrame, spacing: float = 0.5, margin: float = 0):
```